x/ibc/05-port/keeper/keeper.go

```diff
@@ -5,22 +5,23 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/capability"
 	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
 )
 
 // Keeper defines the IBC connection keeper
 type Keeper struct {
```

Comment:
Is this keeper mainly wrapping CapabilityKeeper functionality? Does it need to exist or can it be replaced by CapabilityKeeper?